assembly/src/assembler/instruction/crypto_ops.rs

```diff
@@ -145,27 +145,33 @@ pub(super) fn mtree_set(span: &mut SpanBuilder) -> Result<Option<CodeBlock>, Ass
     // stack: [d, i, R_old, V_new, ...]
 
     // stack: [V_old, R_new, ...] (29 cycles)
-    update_mtree(span, true)
+    update_mtree(span)
 }
 
-/// Appends the MRUPDATE op with a parameter of "true" and stack manipulations to the span block as
-/// required to copy a Merkle tree with root R and update the node in the copied tree at depth d
-/// and index i to value V. The stack is expected to be arranged as follows (from the top):
-/// - depth of the node, 1 element; this is expected to be the depth of the Merkle tree
-/// - index of the node, 1 element
-/// - current root of the tree, 4 elements
-/// - new value of the node, 4 element
+/// Injects [AdviceInjector::MerkleMerge] into the advice provider. The stack is expected to be
+/// arranged as follows (from the top):
```

Comment:
nit: I'd change this to "Creates a new Merkle tree in the advice provider by combining trees with the specified roots. The stack is expected to be arranged as follows (from the top):".

Reply:
Is there a reason this comment was skipped?

Reply:
My terminal CLI didn't render this one; fixed!